playground/ci/find_and_validate_beam_examples.py

```diff
@@ -0,0 +1,64 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+from ci.find_beam_examples import find_examples
+from ci.verify_beam_example import verify_example
+
+
+def main():
+    root_dir = os.getenv("BEAM_ROOT_DIR")
+    if root_dir is None:
+        logging.error("BEAM_ROOT_DIR should be set")
+        # TODO add return from CI step with a fail
+        return
+
+    example_map = []
+    # find all examples and returns file path to each of them
+    examples = find_examples(root_dir)
+    for example in examples:
+        # get sdk based on filepath of example
+        sdk = get_sdk(example)
+
+        # read code from example's file
+        with open(example) as parsed_file:
+            content = parsed_file.read()
+
+        # verify example using code and sdk
+        output = verify_example(content, sdk)
```

Comment:
Can we verify the array from the examples? The client can asynchronously send examples for verification, so I think this will make sense